utils/das_data_sorter.py

```diff
@@ -0,0 +1,65 @@
+import pandas as pd 
+import argparse 
+from math import ceil
+
+# accepts terminal commands
+parser = argparse.ArgumentParser()
+parser.add_argument("--file", help="Input CSV file", action="store")
```

Comment:
I think we need to make this required? Otherwise, how does your script know what file to read?
`parser.add_argument("--file", help="Input CSV file", action="store", required=True)`